### DISCUSSION TITLE
docs/readme/

### DISCUSSION THREAD

**Discussion by giscus** [Yarden-zamir/me#9] [Announcements]
# docs/readme/

https://www.yarden-zamir.com/docs/readme/

**Comment by abhuva** [comment]
Hey, would it be possible to get a more indepth introduction about how you setup/use the obsidian-zola thing? I am using Obsidian publish for nearly 2 years now (and PKM systems for over 10 years) - i really like the idea of having this kind of open/changeable system where other people could also contribute.

**Reply by Yarden-zamir** [reply]
Yes, I will find some time this weekend to write a short blurb to get people started with this
